src/components/redemption/Confirming.js

```diff
@@ -68,12 +58,11 @@ class Confirming extends Component {
   }
 }
 
-const mapStateToProps = (state, ownProps) => {
+const mapStateToProps = (state) => {
   return {
     txHash: state.redemption.txHash,
-    confirmations: state.redemption.confirmations,
-    requiredConfirmations: state.redemption.requiredConfirmations,
-    pollForConfirmationsError: state.redemption.pollForConfirmationsError
+    error: state.redemption.pollForConfirmationsError,
+    btcNetwork: state.redemption.btcNetwork,
```

Comment:
Not blocking, but this should probably be `state.btcNetwork`, and be set from config near startup.

Reply:
Addressed in b553bcc